templates/admin/user/index.html.ep

```diff
@@ -4,7 +4,20 @@
 % content_for 'ready_function' => begin
   $('#users').DataTable( {
     "order": [[0, 'asc']],
-} );
+  } );
+
+$('input[name="role"]:radio').change(function() {
+   var username = $(this).parents('tr').find('.name').text();
+   var role = $(this).attr('id');
+   role = $('label[for="' + role + '"]').text();
+   if (confirm("Sure to make " + username + " a " + $.trim(role) + "?")) {
```

Comment:
the problem is that it "a operator" won't be proper english in any case :)

I'm trying
<pre>
-   if (confirm("Sure to make " + username + " a " + $.trim(role) + "?")) {
+   if (confirm("Are you sure to put " + username + " into role: " + $.trim(role) + "?")) {
</pre>